paper/src/main/java/vg/civcraft/mc/civmodcore/world/locations/chunkmeta/ChunkCoord.java

```diff
@@ -213,9 +212,8 @@ public boolean isChunkLoaded() {
 	 */
 	void minecraftChunkUnloaded() {
 		this.lastUnloadingTime = System.currentTimeMillis();
-		for(ChunkMeta <?> meta : chunkMetas.values()) {
+		for (ChunkMeta<?> meta : chunkMetas.values()) {
```

Comment:
I might have missed this, this is the same issue as above, log and carry on, but don't let exceptions break the loop.

Reply:
Yes, sure, this was fixed too since the similar situation you described for **minecraftChunkLoaded**: https://github.com/CivMC/CivModCore/pull/73/files#diff-bbdcfcd03aae7a13c8bda3bdccc7076e41f1b89f013d55d62c0a5f649b5dcdfbR235